src/net.h

```diff
@@ -111,7 +111,8 @@ struct CSerializedNetMsg
  * connection. Aside from INBOUND, all types are initiated by us.
  *
  * If adding or removing types, please update CONNECTION_TYPE_DOC in
- * src/rpc/net.cpp. */
+ * src/rpc/net.cpp and src/qt/rpcconsole.cpp, as well as the descriptions in
+ * src/qt/guiutil.cpp and src/bitcoin-cli.cpp::NetinfoRequestHandler. */
```

Comment:
nit: A compiler will care of all `switch` statements in `src/qt/guiutil.cpp` :)

Reply:
True. Nevertheless, per element of least surprise, if we list the sites to update we may as well list all of them and save developer time.